protocol/common/message_sender.go

```diff
@@ -504,10 +505,11 @@ func (s *MessageSender) EncodeAbridgedMembershipUpdate(
 func (s *MessageSender) dispatchCommunityChatMessage(ctx context.Context, rawMessage *RawMessage, wrappedMessage []byte) ([]byte, *types.NewMessage, error) {
 
 	newMessage := &types.NewMessage{
-		TTL:       whisperTTL,
-		Payload:   wrappedMessage,
-		PowTarget: calculatePoW(wrappedMessage),
-		PowTime:   whisperPoWTime,
+		TTL:         whisperTTL,
+		Payload:     wrappedMessage,
+		PowTarget:   calculatePoW(wrappedMessage),
+		PowTime:     whisperPoWTime,
+		PubsubTopic: rawMessage.PubsubTopic,
```

Comment:
Can `PubsubTopic`s be a nil / empty value?

Reply:
I'm just wondering what the impact of 2 clients using an older version of `status-go` and one using a version including this change would have. Are they just totally incompatible?

Reply:
A nil/empty pubsub topic will automagically become the DefaultWakuTopic in go-waku.  

Clients will be incompatible the moment a community uses a different shard. If a community continues to use the default waku topic, older and newer clients will keep being able to communicate with each other

Reply:
Thank you for your patience with me on this